conda/exceptions.py

```diff
@@ -405,7 +406,10 @@ def __init__(self, packages, channel_urls=()):
             
             %(channels_formatted)s
             """)
-            packages_formatted = format_list(packages)
+            if type(packages) is not tuple:
+                packages_formatted = "  - " + packages
```

Comment:
same reason here. The `format_list` is designed for new matchspec in 4.4., since conda search passes only one max bad packages at a time, this would be the simplest solution.